content/news/2024/2024-12-21-weekly_15.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Eclipse 4diac Weekly #15"
+slug: "Eclipse 4diac Weekly 15"
+date: 2024-12-21T7:00:00-00:00
+categories: ["4diacweekly"]
+type: "newsitem"
+description: "Update on what development happened across the Eclipse 4diac project in the week from December 13 to December 21, 2024."
```

Comment:
I think it should be from Fr to Fr, so to December 20

Reply:
thx for noting. Just fixed.